dockerfiles/rust-builder/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # DockerHub's official https://hub.docker.com/_/rust
-FROM rust:slim-stretch
+FROM rust:slim-buster
```

Comment:
We should check that this image is signed by Docker.

Reply:
Can we please do this in any other PR? It should be done in CI anyways.

Reply:
https://github.com/paritytech/devops/issues/386